common/src/main/java/me/shedaniel/clothconfig2/gui/widget/SearchFieldEntry.java

```diff
@@ -132,11 +132,11 @@ public Optional<Object> getDefaultValue() {
     
     @Override
     public List<? extends NarratableEntry> narratables() {
-        return List.of(editBox);
+        return Collections.singletonList(editBox);
     }
     
     @Override
     public List<? extends GuiEventListener> children() {
-        return List.of(editBox);
+        return Collections.singletonList(editBox);
```

Comment:
These are the only changes in the bugfix commit, everything else is part of the mapping rename commits.

Reply:
This is very hard to review, mind separating the other changes in another PR?

Reply:
Sure

Reply:
Moved all unrelated changes to #202